ui/package.json

```diff
@@ -17,9 +17,9 @@
     "create:fragment-matcher": "dotenv node src/scripts/queryServer"
   },
   "dependencies": {
-    "@material-ui/core": "^4.5.1",
+    "@material-ui/core": "^4.8.3",
     "@material-ui/icons": "^4.5.1",
-    "@material-ui/lab": "4.0.0-alpha.33",
+    "@material-ui/lab": "4.0.0-alpha.39",
```

Comment:
Let's avoid changing other libraries to minimize the chance of something else breaking.